src/libraries/System.Data.Odbc/src/System/Data/Odbc/Odbc32.cs

```diff
@@ -112,6 +112,7 @@ internal static Exception UnknownOdbcType(OdbcType odbctype)
         }
         internal const string Pwd = "pwd";
 
+        [Conditional("DEBUG")]
         internal static void TraceODBC(int level, string method, ODBC32.SQLRETURN retcode)
```

Comment:
Can we just delete this and the call sites?  We haven't had tracing here this far, I don't see us adding it now.